Sources/PackageModelSyntax/SyntaxEditUtils.swift

```diff
@@ -0,0 +1,225 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift open source project
+//
+// Copyright (c) 2014-2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Basics
+import PackageModel
+import SwiftSyntax
+import SwiftParser
+
+/// Default indent when we have to introduce indentation but have no context
+/// to get it right.
+let defaultIndent = TriviaPiece.spaces(4)
```

Comment:
We do actually have a utility to infer indentation in swift-syntax: https://github.com/apple/swift-syntax/blob/728e2f6d5f9fd4a8e45eab6e52e86ad1519a7e2a/Sources/SwiftBasicFormat/InferIndentation.swift#L19-L27

Reply:
Oh, huh. Should I be worried that this is processing the entire file? I suppose I should thread through an optional indentation value if we're going to do that.

Reply:
I don’t think it should be a huge deal if the user actually invokes `swift package add` and better than just using 4 spaces which does tend to look stupid if the rest of the file is indented with 2 spaces.